src/hrtf/reader.c

```diff
@@ -213,8 +307,11 @@ static struct MYSOFA_HRTF *getHrtf(struct READER *reader, int *err) {
 		} else if (!strcmp(dir->dataobject.name, "Data.Delay")) {
 			*err = getArray(&hrtf->DataDelay, &dir->dataobject);
 		} else {
-			if (!(dir->dataobject.name[0] && !dir->dataobject.name[1]))
-				mylog("UNKNOWN SOFA VARIABLE %s.\n", dir->dataobject.name);
+			if(isNonStandardVariable(dir)) {				
```

Comment:
Also, at this point, we already know that it is a none standard variable (beside the single letters, which still need to be checked)